website/docs/r/namespace.html.md

```diff
@@ -29,3 +29,36 @@ The following arguments are supported:
 ## Attributes Reference
 
 * `id` - ID of the namespace.
+
+## Import
+
+Namespaces can be imported using its `name` as accessor id, e.g.
```

Comment:
Please remove **e.g.** at the end of the sentence. If you want to convey that this is just an example, you can replace e.g. with: **, as an example**. Otherwise, feel free to leave it out.